### ISSUE TITLE
amodel tool bug when installing onnx model

### ISSUE THREAD

**Issue by Michael-Fuu** [ApolloAuto/apollo#15311]
**Describe the bug**
apollo9
amodel install onnx model error

**To Reproduce**
1. wget https://apollo-pkg-beta.cdn.bcebos.com/perception/amodel-0.2.0.tar.gz
2. pip3 install --user amodel-0.2.0.tar.gz
3. wget https://apollo-pkg-beta.cdn.bcebos.com/perception_model/yolox3d_onnx.zip
4. amodel install yolox3d_onnx.zip
""error""
.local/lib/python3.6/site-packages/amodel/model_manage.py", line 224, in _get_install_path_by_meta
    FRAMEWORK_ABBREVIATION[model_meta.framework])
KeyError: 'Onnx'

**Comment by Michael-Fuu** [comment]
it can easily be fixed by modifying amodel/model_manage.py 
`FRAMEWORK_ABBREVIATION = {
  "Caffe": "caffe",
  "PaddlePaddle": "paddle",
  "PyTorch": "torch",
  "TensorFlow": "tf",
  "Onnx":"onnx"}`

**Comment by daohu527** [comment]
Thank you for your reminder. The latest code already contains this configuration. You can ref the https://github.com/ApolloAuto/apollo/discussions/15212 to install amodel

## why
The model installation paths of apollo 8.0 and apollo 9.0 are inconsistent, resulting in amodel **incompatibility**. Therefore, we did not upgrade the amodel version for compatibility, but manually installed the latest version.

In fact, we have more things to consider, such as allowing amodel to be executed outside and inside the container, so we need to install amodel outside the container and then map it to the container. This is also implemented in 8.0

In short, we consider making it easier for users to use tools to manage models, including search, installation, replacement, etc.
There is still a lot of room for improvement, and you are welcome to make more suggestions.
